tests/aws/aws_databases.rs

```diff
@@ -56,6 +65,12 @@ fn deploy_an_environment_with_3_databases_and_3_apps() {
 fn postgresql_deploy_a_working_development_environment_with_all_options() {
     init();
 
+    let span = span!(
```

Comment:
Couldn't it be simplified? Using a function for example?

Reply:
unfortunately, span lives on function they are called